src/test/resources/localstack/setup-sqs.sh

```diff
@@ -13,7 +13,7 @@ aws --endpoint-url=http://localhost:4576 sqs create-queue --queue-name Digital-P
 aws --endpoint-url=http://localhost:4576 sqs create-queue --queue-name Digital-Prison-Services-dev-hmpps_tier_offender_events_queue_d
 aws --endpoint-url=http://localhost:4576 sqs create-queue --queue-name Digital-Prison-Services-dev-hmpps_tier_calculation_complete_queue
 
-aws --endpoint-url=http://localhost:4576 sqs set-queue-attributes --queue-url "http://localhost:4576/queue/Digital-Prison-Services-dev-hmpps_tier_offender_events_queue" --attributes '{"RedrivePolicy":"{\"maxReceiveCount\":\"3\", \"deadLetterTargetArn\":\"arn:aws:sqs:eu-west-2:000000000000:Digital-Prison-Services-dev-hmpps_tier_offender_events_queue_d\"}"}'
+aws --endpoint-url=http://localhost:4576 sqs set-queue-attributes --queue-url "http://localhost:4576/queue/Digital-Prison-Services-dev-hmpps_tier_offender_events_queue" --attributes '{"RedrivePolicy":"{\"maxReceiveCount\":\"1\", \"deadLetterTargetArn\":\"arn:aws:sqs:eu-west-2:000000000000:Digital-Prison-Services-dev-hmpps_tier_offender_events_queue_d\"}"}'
```

Comment:
I don't think this application cares about the redrive policy and it causes confusing logging for any tests which fail to calculate a tier as the message was going back on the queue and being consumed again but without any of the setup in place